### PR TITLE
Update catalan messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
Here we go again, just a few lines, if you would be so kind as to take a look @jordimas @darccio @surfdude29 @rortan134

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -1214,15 +1214,15 @@ msgstr "per <0/>"
 
 #: src/screens/Signup/StepInfo/Policies.tsx:81
 msgid "By creating an account you agree to the <0>Privacy Policy</0>."
-msgstr ""
+msgstr "Creant el compte indiques que estàs d'acord amb la <0>Política de privadesa</0>."
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:1218]: el compte  = un compte
(an = un)

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -325,7 +325,7 @@ msgstr "<0>{0}</0> membres"
 
 #: src/components/dms/DateDivider.tsx:69
 msgid "<0>{date}</0> at {time}"
-msgstr ""
+msgstr "<0>{date}</0> a {time}"
 
 #: src/components/ProfileHoverCard/index.web.tsx:437
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:330]: Jo aquí diria "a les" en comptes de "a"

Assumint que això es mostra així: 30/12/2024 a les 11:12

File: src/locale/locales/ca/messages.po (hunk 1 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po:1218]: Fet, gràcies

File: src/locale/locales/ca/messages.po (hunk 2 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po:330]: correcte

File: src/locale/locales/ca/messages.po [hunk 3]
```diff
@@ -1210,19 +1210,19 @@ msgstr "per <0/>"
...
 
 #: src/screens/Signup/StepInfo/Policies.tsx:81
 msgid "By creating an account you agree to the <0>Privacy Policy</0>."
-msgstr ""
+msgstr "Creant un compte indiques que estàs d'acord amb la <0>Política de privadesa</0>."
 
 #: src/screens/Signup/StepInfo/Policies.tsx:48
 msgid "By creating an account you agree to the <0>Terms of Service</0> and <1>Privacy Policy</1>."
-msgstr ""
+msgstr "Creant un compte indiques que estàs d'acord amb les <0>Condicions del servei/0> i amb la <0>Política de privadesa</0>."
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:1221]:
```suggestion
msgstr "Creant un compte indiques que estàs d'acord amb les <0>Condicions del servei</0> i amb la <0>Política de privadesa</0>."
```
A missing `<` here

File: src/locale/locales/ca/messages.po [hunk 4]
```diff
@@ -1210,19 +1210,19 @@ msgstr "per <0/>"
...
 
 #: src/screens/Signup/StepInfo/Policies.tsx:68
 msgid "By creating an account you agree to the <0>Terms of Service</0>."
-msgstr ""
+msgstr "Creant un compte indiques que estàs d'acord amb les <0>Condicions del servei/0>."
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:1225]:
```suggestion
msgstr "Creant un compte indiques que estàs d'acord amb les <0>Condicions del servei</0>."
```
Also here

**Review by pfrazee — APPROVED** [review]
Thanks all!

File: src/locale/locales/ca/messages.po (hunk 3 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po:1221]: Thanks, I have fixed that in a new PR, this on was approved so fast
https://github.com/bluesky-social/social-app/pull/5735

**Pull request merged**